www/scripts/codecheckerviewer/CommentView.js

```diff
@@ -0,0 +1,272 @@
+// -------------------------------------------------------------------------
+//                     The CodeChecker Infrastructure
+//   This file is distributed under the University of Illinois Open Source
+//   License. See LICENSE.TXT for details.
+// -------------------------------------------------------------------------
+
+define([
+  'dojo/_base/declare',
+  'dojo/dom-construct',
+  'dojo/dom-style',
+  'dojo/topic',
+  'dojo/date/locale',
+  'dojo/store/Memory',
+  'dojo/store/Observable',
+  'dijit/ConfirmDialog',
+  'dijit/Dialog',
+  'dijit/layout/ContentPane',
+  'dijit/form/Button',
+  'dijit/form/SimpleTextarea',
+  'dijit/form/TextBox',
+  'dijit/tree/ObjectStoreModel',
+  'codechecker/HtmlTree',
+  'codechecker/util'],
+function (declare, dom, style, topic, locale, Memory, Observable, ConfirmDialog,
+  Dialog, ContentPane, Button, SimpleTextarea, TextBox, ObjectStoreModel,
+  HtmlTree, util) {
+
+  /**
+   * This function creates a hexadecimal color from a string.
+   */
+  function strToColor(str) {
+    var hash = 0;
+    for (var i = 0; i < str.length; i++)
+       hash = str.charCodeAt(i) + ((hash << 5) - hash);
+
+    var c = (hash & 0x00FFFFFF).toString(16).toUpperCase();
+
+    return '#' + '00000'.substring(0, 6 - c.length) + c;
+  }
+
+  /**
+   * Creates a human friendly relative time ago on the date.
+   */
+  function timeAgo(date) {
+    var delta = Math.round((+new Date - date) / 1000);
+
+    var minute = 60,
+        hour   = minute * 60,
+        day    = hour * 24,
+        week   = day * 7,
+        month  = day * 30
+        year   = day * 365;
+
+    var fuzzy;
+
+    if (delta < 30) {
+      fuzzy = 'just then.';
```

Comment:
just **then**? When? `just now` works better. (Also, that's how GitHub and other such sites show it.)